app/design/frontend/Magento/theme/Magento_Cms/web/css/source/_module.less

```diff
@@ -73,28 +75,31 @@
         .baner-link {
 
 
-        color: @color-elogic-black;
-        text-decoration: none;
+            color: @color-elogic-black;
+            text-decoration: none;
             .lib-font-size(16) ;
-        border-bottom: 1px solid @color-elogic-black;
+            border-bottom: 1px solid @color-elogic-black;
 
-            &:hover{
+            &:hover {
                 color: @color-elogic-gray;
-                border-bottom: 1px solid @color-elogic-gray ;
+                border-bottom: 1px solid @color-elogic-gray;
 
             }
+        }
     }
-}
+
     //----------------------------------------------------
-         .categories{
-             text-align: center;
-         }
-       .category-clocks,
-       .category-newLight {
-           background-color: @color-background;
-           padding: 0 30px 30px 30px;
-       }
-        .category-content {
+    .categories {
+        text-align: center;
+    }
+
+    .category-clocks,
+    .category-newLight {
+        background-color: @color-background;
+        padding: 0 30px 30px 30px;
```

Comment:
check these variables for paddings/margins:
@indent__xs;
@indent__s;
@indent__base;
@indent__l;
@indent__xl;